packages/smithy-http/src/smithy_http/testing/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Shared utilities for smithy-python functional tests."""
```

Comment:
"functional testing" may not be quite how we want to present this. They're general tools for testing any generated clients that have an HTTP-based transport.
```suggestion
"""Shared utilities for testing smithy-python clients with an HTTP transport."""
```